reactos/dll/win32/browseui/shellbrowser.cpp

```diff
@@ -1832,6 +2010,9 @@ HRESULT STDMETHODCALLTYPE CShellBrowser::Exec(const GUID *pguidCmdGroup, DWORD n
             case 0x12:
                 // refresh on toolbar clicked
                 return S_OK;
+            case 0x26:
```

Comment:
I'm not even sure those values are really correct (it's sometimes hard and confusing to understand API Monitor traces)